### DISCUSSION TITLE
UEFI or BIOS installation notification

### DISCUSSION THREAD

**Discussion by Laminaat** [rhinstaller/anaconda#4461] [Ideas]
Recently I fully switched from Windows 10/11 to Fedora (36 at the time). One thing that went wrong during the initial installation of Fedora was that I installed it as a BIOS based installation. Due to some later troubleshooting I figured this out and came to the realisation that the motherboard was setup in a UEFI/BIOS (default) mode instead of just UEFI which worked fine for Windows. I would recommend that the Anaconda installer would notify the user in a separate step if the install will be performed based on BIOS or UEFI mode. And maybe with some information about the implications and what can be done about it.

The 'Storage options' information blurb does say something about UEFI/BIOS but I would make it a step in the installation. Changing an UEFI setting before installing Fedora seems a lot easier then doing a reinstall later or changing the bootloader on an existing installation when you find out on your own.

**Comment by jkonecny12** [comment]
Interesting idea. Thanks for the feedback.
